src/main/java/io/github/kawamuray/wasmtime/Memory.java

```diff
@@ -20,6 +20,8 @@ public Memory(Store store, MemoryType type) {
 
     public native int size();
 
+    public native int grow(int page);
```

Comment:
I think the argument name should be one of `pages` (plural), `delta` (by rust's API) or `delta_pages` (most understandable?)